src/main/resources/application.yml

```diff
@@ -45,18 +45,10 @@ app:
     version: ^project.version^
   verifiableCredentials:
     durationDays: 90
-    schema2210Url: https://f1c82785-5598-41c7-a083-01a8e1a80e19.mock.pstmn.io/ctxsd
+    schema2210Url: https://sdfactory.int.demo.catena-x.net/context/sd-document-v2210
     gaia-x-participant-schema: https://registry.lab.gaia-x.eu/development/api/trusted-schemas-registry/v2/schemas
     gaia-x-service-schema: https://gaia-x.gitlab.io/gaia-x-community/gaia-x-self-descriptions/service/ontology.json
```

Comment:
hey @dvasunin, we should avoid consortia related references (urls, environments etc.). Would also strongly suggest to leverage helm values.yaml for apps configuration of such environment related variables adhereing to [TRG 5.06](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-06)

Reply:
@tomaszbarwicki  we are looking into it.

Reply:
@dvasunin Remove this hardcoded URL. I will suggest left as empty value and comment over for user information.

Reply:
done